ocs_ci/ocs/platform_nodes.py

```diff
@@ -922,7 +937,7 @@ def attach_rhel_nodes_to_upi_cluster(self, node_list):
             rhel_pod_obj.exec_cmd_on_node(
                 host,
                 pem_dst_path,
-                f'sudo mv {os.path.join("/tmp", repo_file)} {repo_dst_path}',
+                f'sudo mv {os.path.join("/tmp", repo_file)} {constants.YUM_REPOS_PATH}',
```

Comment:
Same as above, the `/tmp` here and few lines above could be probably replaced by constants `POD_UPLOADPATH` or `RHEL_TMP_PATH` (but I'm ok to keep it as it is in this PR, if there will be no other changes).